packages/@okta/vuepress-site/docs/reference/api/user-types/index.md

```diff
@@ -308,7 +308,7 @@ HTTP/1.1 204 No Content
 
 ## Specify the User Type of a new user
 
-The [Create User](/docs/reference/api/users/#create-user-with-non-default-user-type) operation accepts a type specification as part of the request body. The specification is a map, but currently the only key permitted is `id`. The type specification is also added to the [User object](/docs/reference/api/users/#user-object), but after user creation the type may only be updated by an administrator on a [full replace of an existing user](/docs/reference/api/users/#update-user), not a partial update.
+The [Create User](/docs/reference/api/users/#create-user-with-non-default-user-type) operation accepts a type specification as part of the request body. The specification is a map, but currently, the only key permitted is `id`. The type specification is also added to the [User object](/docs/reference/api/users/#user-object), but after user creation, the type can only be updated by an administrator on a [full replace of an existing user](/docs/reference/api/users/#update-user), not a partial update.
```

Comment:
the comma after currently isn't necessary

also

but after user creation, only an administrator can update the type by doing a [full replace of an existing user](.......), not a partial update.